mobly/controllers/android_device.py

```diff
@@ -123,7 +123,7 @@ def destroy(ads):
   for ad in ads:
     try:
       ad.services.stop_all()
-    except:
+    except Exception:
```

Comment:
Fix according to Flake8 rule [E722](https://www.flake8rules.com/rules/E722.html)
```
Do not use bare except, specify exception instead
```